core/storage/config/gae_models.py

```diff
@@ -55,22 +62,30 @@ class ConfigPropertyModel(base_models.VersionedModel):
 
     @staticmethod
     def get_deletion_policy():
+        # type: () -> base_models.DELETION_POLICY
         """ConfigPropertyModel is not related to users."""
         return base_models.DELETION_POLICY.NOT_APPLICABLE
 
     @staticmethod
     def get_model_association_to_user():
+        # type: () -> base_models.MODEL_ASSOCIATION_TO_USER
         """Model does not contain user data."""
         return base_models.MODEL_ASSOCIATION_TO_USER.NOT_CORRESPONDING_TO_USER
 
     @classmethod
     def get_export_policy(cls):
+        # type: () -> Dict[Text, base_models.EXPORT_POLICY]
         """Model doesn't contain any data directly corresponding to a user."""
         return dict(super(cls, cls).get_export_policy(), **{
             'value': base_models.EXPORT_POLICY.NOT_APPLICABLE
         })
 
-    def commit(self, committer_id, commit_cmds):
+    # TODO(#13523): Change 'commit_cmds' to TypedDict to remove Any here.
+    # We have ignored [override] here because the signature of this method
+    # doesn't match with VersionedModel.commit().
+    # https://mypy.readthedocs.io/en/stable/error_code_list.html#check-validity-of-overrides-override
+    def commit(self, committer_id, commit_cmds): # type: ignore[override]
```

Comment:
Confirming that this specific override is OK. (Again, this is not a general approval for other overrides.)

Reply:
Thanks!